styles.css

```diff
@@ -1,54 +1,52 @@
 #title {
-    text-align: center;
+  text-align: center;
 }
 
 #survey-form {
-    text-align: center;
-    width: 40%;
-    background-color: grey;
-    margin: auto;
-    padding: 20px;
-    border-color: aqua;
-    border-radius: 10px;
-
+  text-align: center;
+  width: 50%;
+  background-color: green;
+  margin: auto;
+  padding: 20px;
+  border-color: aqua;
+  border-radius: 10px;
 }
 
 p {
-    text-align: center;
+  text-align: center;
 }
 
-Label {
-    color:#388080;
-    background-color:#eceff1; 
-    width: 25%;
-    padding: 4px;
-    margin: 10px;
-    
+label {
+  color: #388080;
+  background-color: #eceff1;
+  width: 25%;
+  padding: 4px;
+  margin: 10px;
 }
 
-#name, #email, #number{
-    width: 60%;
+#name,
+#email,
+#number {
+  width: 60%;
 }
 
-#submit{
-    width: 60%;
-    background-color: #388080;
-    color: white;
-    border-color:#388080;
-    border-radius: 10px;
+#submit {
+  width: 60%;
+  background-color: #388080;
+  color: white;
+  border-color: #388080;
+  border-radius: 10px;
```

Comment:
could you please change the border color to black as we need a darker color to show the border around the submit button and it would be even nicer if you please change your border radius to 15px as we need a more rounded shaped button